File: FlexServ Backend/SpringBoot/AuthService/src/main/java/com/flexserv/dto/response/LoginResponse.java
package com.flexserv.dto.response;

import com.flexserv.entity.Role;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
public class LoginResponse {

    private Long userId;

    private String name;

    private Role role;

    private String token;

    public LoginResponse(Long userId, String name, Role role) {
        this.userId = userId;
        this.name = name;
        this.role = role;
    }
}

File: FlexServ Backend/DotNet/AuthService/Dtos/LoginResponse.cs
using AuthService.Models;

namespace AuthService.Dtos
{
    public class LoginResponse
    {
        public long Id { get; set; }
        public string Name { get; set; } = string.Empty;
        public Role Role { get; set; }

        public LoginResponse() { }

        public LoginResponse(long id, string name, Role role)
        {
            Id = id;
            Name = name;
            Role = role;
        }
    }
}
